src/workers/notifications/utils.ts

```diff
@@ -92,6 +92,9 @@ export async function articleNewCommentHandler(
   if (!comment) {
     return;
   }
+  if (comment.flags.vordr) {
```

Comment:
Just in case

```suggestion
    if (comment.flags?.vordr) {
```

Reply:
hopefully with strict TS we won't have to catch these kinds of issues manually 🤞

Reply:
Added